src/aap_eda/core/validators.py

```diff
@@ -45,11 +45,7 @@ def check_if_de_exists(decision_environment_id: int) -> int:
     try:
         de = models.DecisionEnvironment.objects.get(pk=decision_environment_id)
         if de.eda_credential_id:
-            models.EdaCredential.objects.get(pk=de.eda_credential_id)
```

Comment:
@hsong-rh What happened to this check why did it get removed. If we have a bogus id there we still need to catch it during validation

Reply:
Never mind I see that other function is doing the check 👍